mail/postfix/Makefile

```diff
@@ -64,16 +74,26 @@ define Package/postfix/config
 			default y
 			help
 			  Implements support for SQLite3 DB
+		config POSTFIX_PGSQL
+			bool "PostgreSQL support"
+			default n
+			help
+			  Implement support for PostgreSQL
+		config POSTFIX_PCRE
+			bool "PCRE support"
+			default n
+			help
+			  Implement support for Perl Compatible Regular Expressions
 		config POSTFIX_EAI
 			bool "SMTPUTF8 support"
 			default n
 			help
-				Enable Postfix support for Email Address Internationalization
-				(EAI) as defined in RFC 6531 (SMTPUTF8 extension), RFC 6532
-				(Internationalized email headers) and RFC 6533
-				(Internationalized delivery status notifications).
-				Since version 3.0, Postfix fully supports UTF-8 email
-				addresses and UTF-8 message header values.
+			  Enable Postfix support for Email Address Internationalization
```

Comment:
yes I know, is tab-tab-space-space like other help information on the file, maybe should I make tab-tab for all the help statements ?